sycl/include/sycl/ext/oneapi/experimental/detail/properties/graph_properties.hpp

```diff
@@ -0,0 +1,67 @@
+//==----------- queue_properties.hpp --- SYCL queue properties -------------==//
```

Comment:
needs updating to "graph"

Reply:
fixed in https://github.com/intel/llvm/pull/15647/commits/7ceb2b88a8b3e9e75b6c24158f029eb24f0180f6